### ISSUE TITLE
foreign debootstrap fails on kernel 5.13 with: /bin/sh: 0: Illegal option --

### ISSUE THREAD

**Issue by obbardc** [go-debos/debos#287]
In debian unstable with kernel 5.13 from experimental, debos fails when debootstrapping  a foreign arch.

test recipe:
```
architecture: arm64
actions:
  - action: debootstrap
    suite: buster
    components:
      - main
    mirror: https://deb.debian.org/debian
    variant: minbase
  - action: pack
    file: ospack-sid-arm64.tar.gz
```

output:
```
$ debos --debug-shell debian-base-ospack-foreign.yaml
...
2021/09/01 10:46:20 Debootstrap (stage 2) | /bin/sh: 0: Illegal option --
...
2021/09/01 10:46:20 Action `debootstrap` failed at stage Run, error: exit status 2
```

testing the command in the debug shell:
```
2021/09/01 10:46:20 >>> Starting a debug shell

bash-5.1# chroot /scratch/root /debootstrap/debootstrap --no-check-gpg --second-stage --components=main
/bin/sh: 0: Illegal option --

bash-5.1# chroot root /debootstrap/debootstrap --help
/bin/sh: 0: Illegal option --

bash-5.1# root/debootstrap/debootstrap --second-stage
W: Unable to read /etc/apt/apt.conf.d/ - DirectoryExists (2: No such file or directory)
```

**Comment by obbardc** [comment]
Possibly https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=993658 is related

**Comment by obbardc** [comment]
yup; it was #993658 which is fixed in qemu 1:6.1+dfsg-5
